shared/src/commonMain/kotlin/org/hyperskill/app/study_plan/widget/view/StudyPlanWidgetViewStateMapper.kt

```diff
@@ -100,13 +100,14 @@ class StudyPlanWidgetViewStateMapper(private val dateFormatter: DateFormatter) {
                         null -> StudyPlanWidgetViewState.SectionItemState.IDLE
                     },
                     progress = null, // TODO: add data with new activities API
-                    formattedProgress = null // TODO: add data with new activities API
+                    formattedProgress = null, // TODO: add data with new activities API
```

Comment:
There is information In [this](https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-695/Shared-Support-new-fields-from-the-LearningActivity-API#focus=Comments-74-226291.0-0) comment, that we should implement 100% (for skipped and completed topics) and 0% (in other cases) progress

Reply:
@vladkash we are fetching learning activities only in the `TODO` state, so it's impossible to have completed progress, see "[comment](https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-695/Shared-Support-new-fields-from-the-LearningActivity-API#focus=Comments-74-226300.0-0)"